unit-tests.sh

```diff
@@ -26,16 +24,16 @@ function run_unit_tests() {
     fi
 }
 
-echo "running unit tests with sqlite in memory"
-# tests with sqlite
-run_unit_tests
+if [ -z "$STORAGE" ] && command -v sqlite > /dev/null 2>&1; then
+    # tests with sqlite
+    echo "running unit tests with sqlite in memory"
+    CONFIG_FILE=config.toml
+    run_unit_tests
+fi
 
-if netstat -nltp 2>/dev/null | grep "$POSTGRES_PORT" >/dev/null; then
+if [ "$STORAGE" == "postgres" ]; then
     # tests with postgres
-    export INSIGHTS_RESULTS_AGGREGATOR__TESTS_DB="aggregator"
-    export INSIGHTS_RESULTS_AGGREGATOR__TESTS_DB_ADMIN_PASS="password"
-    echo "running unit tests with postgres on port $POSTGRES_PORT"
+    echo "running unit tests with postgres"
+    CONFIG_FILE=config-devel.toml
     run_unit_tests
-else
-    echo "postgres on port $POSTGRES_PORT is not running, skipping unit tests with postgres"
```

Comment:
Maybe we need this code so that `make before_commit` can run tests on postgres in case postgres is in the system

Reply:
I propose to explicitly specify if you have running postgres locally. I made this change because  `registry.redhat.io/rhel8/go-toolset:1.13` doesn't have `netstat`.